src/gf.c

```diff
@@ -2028,6 +2028,12 @@ JL_DLLEXPORT jl_value_t *jl_fptr_sparam(jl_value_t *f, jl_value_t **args, uint32
     }
 }
 
+JL_DLLEXPORT jl_callptr_t jl_fptr_args_addr = &jl_fptr_args;
+
+JL_DLLEXPORT jl_callptr_t jl_fptr_const_return_addr = &jl_fptr_const_return;
+
+JL_DLLEXPORT jl_callptr_t jl_fptr_sparam_addr = &jl_fptr_sparam;
```

Comment:
use `jl_invoke_api` instead?

Reply:
:+1: unfortunately there is one place in jitlayers.cpp that writes to it as well, so we would need to add a setter method.